code/modules/clothing/gloves/tacklers.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/clothing/gloves/tackler
-	name = "gripper gloves"
-	desc = "Special gloves that manipulate the blood vessels in the wearer's hands, granting them the ability to launch headfirst into walls."
+	name = "Enhanced Retrieval gloves"
```

Comment:
```suggestion
	name = "enhanced retrieval gloves"
```
we don't capitalize names unless it's some brand it looks stupid

Reply:
go easy on contributors man wtf :/